docs/userGuide/syntax/panels.md

```diff
@@ -104,6 +104,14 @@ plain text ...
 </variable>
 </include>
 
+<box background-color="#C51E3A" color="white">
+  
+  :bulb: Seamless panels inherit the background colour and text colour of any parents!
+  <br/>
+  <panel type="seamless" header="This is an example seamless panel" popup-url="/userGuide/syntax/extra/loadContent.html">
```

Comment:
Why the popup-URL ?

Reply:
I'm just making sure that all buttons follow the seamless behaviour when testing, I'll remove from the final example!